conf/application.conf

```diff
@@ -3,7 +3,7 @@ play.http.port=9000
 
 auth.url="http://localhost:8080"
 auth.callback="http://localhost:9000/callback"
-auth.secret="set_me"
+auth.secret="[set me]"
```

Comment:
I know this isn't really your change, but is it actually necessary to set this secret? I think it gets overridden in all of the deployed environments, and it doesn't matter what you set it to in development.

Maybe setting it to `dev-environment-placeholder` might make it clearer, so that it's obvious that it should only be used in development and you don't have to change it when you set up the project?

Reply:
In the local environment it needs to be set to the tdr client crendentials secret that is setup in the local keycloak application.

Reply:
Should it be read from an environment variable, then? Otherwise (as far as I can tell) you'll have changes that you have to avoid committing.